packages/cozy-stack-client/src/TriggerCollection.js

```diff
@@ -122,7 +122,8 @@ class TriggerCollection extends DocumentCollection {
    */
   async find(selector = {}, options = {}) {
     const { worker, type, ...rest } = selector
-    const hasOnlyWorkerAndType = Object.keys(rest).length === 0
+    const hasOnlyWorkerAndType =
+      Object.keys(rest).length === 0 && !options.partialFilter
```

Comment:
Hmm I don't think so, `hasOnlyWorkerAndType` is true if there is no other keys than `worker` and `type` in selector AND no partialFilter.
Reversely, if there is another key or a partialFilter,  `hasOnlyWorkerAndType` is false

Reply:
`and NO partialFilter` that part made me think that it is not accurate anymore.

Is it possible that there is no `worker` and `type` in selector but a partial filter is present ?

Reply:
> Is it possible that there is no worker and type in selector but a partial filter is present ?

Yes it can. And in this case, `hasOnlyWorkerAndType` is false, as there is a partialFilter